src/components/SearchResults.astro

```diff
@@ -0,0 +1,137 @@
+---
+import { fetchSiteConfig } from "@/utilities/queries";
+
+const siteConfig = await fetchSiteConfig();
+
+const apiKey = siteConfig?.searchAccessKey;
+const affiliate = siteConfig?.searchAffiliate;
+
+const { query } = Astro.props;
+
+const response = await fetch(
```

Comment:
This `fetch` doesn't appear to be kicking off when the page loads. [Example search](https://federalist-0e46ca58-4e05-4c83-b268-7c7844e8d86e.sites.pages-dev.cloud.gov/preview/cloud-gov/pages-site-gantry/feat-search-view-component/search/?utf8=%E2%9C%93&affiliate=demo-gantry&query=hello)